### PR TITLE
Islam

### PR DESCRIPTION
**Pull request by islam95**

### CODE REVIEW

File: src/components/Section1.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
```
Comment (kabaros) [inline src/components/Section1.js]: Naming of file and component is not good. You can call these `Home`, `About` and `Albums` (once you start calling things `component1`, `component2` then it's a sign you have to rethink them). 

Also if there is no state, then just use a simple functional component

File: src/components/Albums/Albums.js [hunk 2]
```diff
@@ -1,16 +1,47 @@
...
+        )}
+      </div>
+    );
   }
 }
 
-Albums.propTypes = {
-
-};
+Albums.propTypes = {};
```
Comment (kabaros) [inline src/components/Albums/Albums.js]: don't have empty proptypes

File: src/components/Albums/Albums.js [hunk 3]
```diff
@@ -1,16 +1,47 @@
...
 class Albums extends Component {
+  constructor() {
+    super();
+    this.state = {
+      albums: []
+    };
+  }
+
+  componentDidMount() {
+    fetch(
```
Comment (kabaros) [inline src/components/Albums/Albums.js:14]: instead of using promises (`.then`), google and read about `async/await` and use it instead.

Comment (kabaros) [reply src/components/Albums/Albums.js:14]: what happens if an error happens? if the call to get the albums fails - handle the exception (catch) and display an error. You can test it by switching network off in the Developers tools > Network tab, or just change the url to a wrong one.

**Review by kabaros — CHANGES_REQUESTED** [review]
Also give unit tests a try to practice them

File: src/components/Albums/Albums.js (hunk 3 above)
Comment (islam95) [reply src/components/Albums/Albums.js:14]: Thank you Mozafar, I like such comments in my code. When someone just says your code is good is not good for me :)

**Pull request closed without merging**